index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="styles.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=<device-width>, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap" rel="stylesheet">
+  <title>Luis's Portfolio</title>
+</head>
+<body>
+  <header>
+    <div id="welcome" href="#">Welcome</div>
+    <nav id="nav"><a href="#">
+      <img src="hamburger.png"></a>
+    </nav>
+  </header>
+  <section>
+    <div id="background">
+      <h1 class="text-container" id="hisection">
+        Hey there. I’m Luis
+        <h2 class="text-container" id="sdsection">
+          I’m a software developer
+        </h2>
+      </h1>
+      <p id="description"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.</p></div>
+      <ul>
+        <li><a href="#"><img src="twitter.svg"></a></li>
+        <li><a href="#"><img src="linkedin.svg"></a></li>
+        <li><a href="#"><img src="medium.svg"></a></li>
+        <li><a href="#"><img src="github.svg"></a></li>
+        <li><a href="#"><img src="angellist.svg"></a></li>
+      </ul>
```

Comment:
- [ ] KIndly align your social icons to be on the same line so that it matches the figma design just as the previous reviewer has suggested 👍🏽 

Your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/212426379-0c34c508-8b59-4875-a662-16be6ab8cc3e.png)

Figma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/212426562-f5b2d061-30da-4ed5-9cb9-57054befeb8e.png)